src/Uno.Toolkit.UI/Behaviors/InputExtensions.cs

```diff
@@ -177,5 +237,37 @@ private static void OnUIElementKeyUp(object sender, KeyRoutedEventArgs e)
 				_ => default,
 			};
 		}
+
+#if __ANDROID__
+		private static ImeAction GetImeActionFromReturnType(ReturnType returnType)
+		{
+			switch (returnType)
+			{
+				case ReturnType.Go: return ImeAction.Go;
+				case ReturnType.Search: return ImeAction.Search;
+				case ReturnType.Send: return ImeAction.Send;
+				case ReturnType.Done: return ImeAction.Done;
+				case ReturnType.Default:
+				case ReturnType.Next:
+				default: return ImeAction.Next;
+			}
+		}
+#endif
+
+#if __IOS__
+		private static UIReturnKeyType GetReturnKeyTypeFromReturnType(ReturnType returnType)
+		{
+			switch (returnType)
+			{
+				case ReturnType.Go: return UIReturnKeyType.Go;
+				case ReturnType.Search: return UIReturnKeyType.Search;
+				case ReturnType.Send: return UIReturnKeyType.Send;
+				case ReturnType.Done: return UIReturnKeyType.Done;
+				case ReturnType.Default:
+				case ReturnType.Next:
+				default: return UIReturnKeyType.Next;
```

Comment:
Same here, should it be UIReturnKeyType.Next?

Reply:
https://developer.apple.com/documentation/uikit/uireturnkeytype
`.Default` and default(else-case) should map to `UIReturnKeyType.Default`; and
only `.Next` to `...Next`